core/src/main/java/io/confluent/kafka/schemaregistry/storage/KafkaStore.java

```diff
@@ -272,10 +272,11 @@ private void createSchemaTopic() throws StoreInitializationException {
   }
 
   static List<String> brokersToEndpoints(List<Broker> brokers) {
-    List<String> endpoints = new LinkedList<String>();
+    List<String> endpoints = new LinkedList<>();
     for (Broker broker : brokers) {
       for (EndPoint ep : JavaConversions.asJavaCollection(broker.endPoints())) {
-        endpoints.add(ep.connectionString());
+        String hostport = ep.host() == null ? ":" + ep.port() : Utils.formatAddress(ep.host(), ep.port());
+        endpoints.add(ep.securityProtocol() + "://" + hostport);
```

Comment:
Agreed that it's awkward to split the security protocol into a separate config.  What's weird is that the bootstrap server param actual [does seem to accept it (not complain) but will ignore it](https://github.com/apache/kafka/blob/trunk/clients/src/main/java/org/apache/kafka/common/utils/Utils.java#L52-L54).

Didn't we have this same problem before KIP-103?